rerun

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I agree, it would be better to put `rerun` in tests/bin unless and until it is being used elsewhere. We will just confuse people by dropping a binary in the root of the repo.